projects/angular/src/forms/datepicker/date-container.ts

```diff
@@ -140,6 +168,29 @@ export class ClrDateContainer extends ClrAbstractContainer implements AfterViewI
     }
   }
 
+  @Input('rangeOptions')
+  set rangeOptions(rangeOptions) {
```

Comment:
```suggestion
  @Input()
  set rangeOptions(rangeOptions) {
```

input alias not needed